test-framework/maven/src/main/java/io/quarkus/maven/it/MojoTestBase.java

```diff
@@ -121,7 +121,7 @@ public static void assertThatOutputWorksCorrectly(String logs) {
         assertThat(logs.contains(infoLogLevel)).isTrue();
         Predicate<String> datePattern = Pattern.compile("\\d{4}-\\d{2}-\\d{2}\\s\\d{2}:\\d{2}:\\d{2},\\d{3}").asPredicate();
         assertThat(datePattern.test(logs)).isTrue();
-        assertThat(logs.contains("cdi, resteasy, servlet, smallrye-context-propagation, undertow-websockets")).isTrue();
+        assertThat(logs.contains("cdi, resteasy, smallrye-context-propagation, undertow-websockets")).isTrue();
```

Comment:
The name of the extension should not contain `undertow` (the feature here but same with the module name). So... we will probably have to create another extension? Maybe we could keep the Undertow one around for a couple of versions?

Reply:
Ah or even better, we keep `undertow-websockets` around but it's just a shell dependency with dependencies to the new extension and `undertow`.

And we mark it as deprecated and `unlisted`.

Reply:
Technically it still is Undertow, just Undertow websockets instead of Undertow Servlet, but I think it makes sense to just call it websockets to avoid confusion.